docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,214 +1,169 @@
-  <ScopedBlock scope={["oss", "enterprise"]}> This guide assumes that you will
-  run Teleport's Windows Desktop Service on a dedicated host. To install Desktop
-  Access into an existing Teleport instance running other services, see the
-  [Manual Setup](manual-setup.mdx) guide for Desktop Access.</ScopedBlock>
-- An Active Directory domain, configured for LDAPS (Teleport requires an
-  encrypted LDAP connection). Typically this means installing
-  [AD CS](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/)
+- (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-## Step 1/2. Install Teleport
+- A Linux server to run the Teleport Desktop Access service on.
+  You can reuse an existing server running any other Teleport instance.
 
-### Set up the Teleport Auth and Proxy Services
+- (!docs/pages/includes/tctl.mdx!)
 
-(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+## Step 1/4. Prepare Windows
 
-## Step 2/2. Run the discovery wizard
+### Import the Teleport root certificate
 
-In your web browser, access the teleport Web UI at <ScopedBlock scope={["oss", "enterprise"]}>
-`teleport.example.com`</ScopedBlock><ScopedBlock scope={["cloud"]}>
-`mytennant.teleport.sh`</ScopedBlock>. Click on your user name at the top right
-and select **Manage Access**, Select "Desktop" from the main menu, then **NEXT**:
+Use `tctl` to export the Teleport root certificate:
 
-If you already have Active Directory installed, skip to the next step. Otherwise,
-copy and paste the first command provided into a Windows PowerShell window. If
-you aren't already running AD Certificate services, copy and paste the second
-command after the first one completes and the server restarts:
+```code
+$ tctl auth export --type=windows > teleport.cer
+```
 
-![Install Active Directory](../../img/desktop-access/install-active-directory.png)
+Copy this certificate to your Windows system, if you didn't run `tctl` from there.
 
-Once the server is restarted from one or both command executions, click **NEXT**.
+### Install the Teleport service for Windows
 
-Copy and paste the provided command into a Windows PowerShell window to download
-and run the 'configure Active Directory' script:
+From the Windows system, download the [Teleport service installer](#LinkToCome) {/*Rename as needed */}
+and run it. When prompted, select the Teleport certificate file from the previous step.
+Once complete, reboot the system.
 
-![Configure Active Directory](../../img/desktop-access/configure-active-directory.png)
+## Step 2/4. Install the Teleport Desktop Service
 
-Click **NEXT**.
+On the Linux host where you will run the Desktop Service, install Teleport (if
+you plan on running the Windows protocol on the same host as an existing Teleport
+service, you can skip this step):
 
-The PowerShell script will output a Teleport configuration block. Copy this
-block to a temporary location. Click **Next**.
+(!docs/pages/includes/install-linux.mdx!)
```

Comment:
This partial has its own intro paragraph that conflicts with the one in this guide:

<img width="907" alt="image" src="https://user-images.githubusercontent.com/25250668/214699702-e1cb3681-3871-4879-b872-69825feff599.png">

Reply:
Imported a commit from another PR that makes the partial more compatible with preceding content from the docs it's called in.